cpp_build/src/parser.rs

```diff
@@ -0,0 +1,558 @@
+use cpp_common::{Class, Closure, Macro, RustInvocation};
```

Comment:
I think we're going to have to do some level of custom parsing to keep working without a cpp fork, but I think we might be able to do it in a cleaner way. I'm probably going to take a bit to think about this.